index.html

```diff
@@ -1,11 +1,41 @@
-Ki Vera Luna Pardillo
+<!DOCTYPE html>
+<html>
```

Comment:
:+1:

Bonus, to help users who use screen readers, we can also add a \`lang\` attribute to the \`html\` tag, which will tell screen readers which language the page is in. Eg for English:

```html
<html lang="en">  
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/html#accessibility_concerns